ts/editor/StickyBadge.svelte

```diff
@@ -37,20 +35,31 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 <span class:highlighted={active} on:click|stopPropagation={toggle}>
     <Badge
         tooltip="{tr.editingToggleSticky()} ({getPlatformString(keyCombination)})"
-        widthMultiplier={0.7}
-        --icon-align="text-top">{@html icon}</Badge
+        widthMultiplier={0.7}>{@html stickyIcon}</Badge
     >
 </span>
 
 <style lang="scss">
     span {
-        opacity: 0.4;
-
+        cursor: pointer;
+        opacity: 0;
         &.highlighted {
             opacity: 1;
         }
+    }
+    :global(.editor-field) {
+        &:focus-within,
```

Comment:
Applying the hover/focus boolean to this one seems a bit more tricky because `StickyBadge` is located in `NoteCreator`. Can I move it down to `NoteEditor`?

Reply:
Nevermind, I just found out about slot props :)

But I think I found a Svelte bug with https://github.com/ankitects/anki/pull/2002/commits/2806c617b31fa3f7475150fc3397278c5f9e47db.

The `active` prop of `StickyBadge` resets to false as soon as the new `visible` prop is updated. This makes the sticky badge disappear on mouseleave/blur, regardless of its back-end value. I can't for the life of me figure out what I've done wrong to cause this issue.